doc/themes/the-things-stack/layouts/partials/distribution-badges.html

```diff
@@ -0,0 +1,11 @@
+<div class="buttons">
+	{{ range . }}
+		{{ $distribution := index site.Data.distributions . }}
+		<a href="/download" class="button is-small is-short" style="color:{{ index $distribution "color" }};border-color:{{ index $distribution "color" }};">
+			<span class="icon">
+		    	<img src="{{ index $distribution "icon" }}"></img>
+		    </span>
+		    <span class="has-text-weight-bold">{{ . }}</span>
```

Comment:
- Inconsistent use of tabs and spaces
- Use `relURL` (like we do in check-icon: `src="{{ "img/check.svg" | relURL }}"`)